src/views/Articles/Posts.jsx

```diff
@@ -1,73 +1,124 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import Moment from 'react-moment';
 import Spinner from '../Article/Spinner';
-import PostFeed from '../../components/Articles/PostFeed';
-import { getPosts } from '../../redux/actions/postActions';
+import { getPages, getNextPages } from '../../redux/actions/postActions';
+import articlesUrl from '../../services/api';
 import '../../styles/App.css';
+import '../../styles/pagination.css';
+import PaginationContainer from '../Pagination/PaginationContainer';
 
 
 
 class Posts extends Component {
+  state = {
```

Comment:
Hello,
You should also ensure this state comes after the componentDidMount method to solve the warnings that arise

Reply:
The issue is with setting state in the componentDidMount() lifecycle method. @Ogutu-Brian

Reply:
@Ogutu-Brian  and @IssaIan  solved